src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
     private readonly IReadOnlyDictionary<string, Language> _languages = new ReadOnlyDictionary<string, Language>(new Dictionary<string, Language>(StringComparer.OrdinalIgnoreCase)
     {
         ["af"] = new("Afrikaans", "Afrikaans", "af", "afr"),
-        ["sq"] = new("Albanian", "Shqip", "sq", "sqi"),
         ["am"] = new("Amharic", "አማርኛ", "am", "amh"),
         ["ar"] = new("Arabic", "العربية", "ar", "ara"),
-        ["hy"] = new("Armenian", "Հայերեն", "hy", "hye"),
+        ["as"] = new("Assamese", "অসমীয়া", "as", "asm", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ay"] = new("Aymara", "aymar aru [ajmaɾ ˈaɾu]", "ay", "aym", TranslationServices.Google),
```

Comment:
Native name should be "Aymar aru".
```suggestion
        ["ay"] = new("Aymara", "Aymar aru", "ay", "aym", TranslationServices.Google),
```

Reply:
fixed